src/test/ui/feature-gate-i128_type2.rs

```diff
@@ -10,20 +10,20 @@
 
 // gate-test-i128_type
 
```

Comment:
Why not remove the `i128_type` tests?

Reply:
I left this one as a test for the repr128 gate.

Reply:
Oh, it looks like there is already a test for that.